test/automation/src/sql/notebook.ts

```diff
@@ -95,4 +114,26 @@ export class NotebookToolbar {
 		const kernelDropdownValue = `${NotebookToolbar.toolbarSelector} select[id="kernel-dropdown"][title="${kernel}"]`;
 		await this.code.waitForElement(kernelDropdownValue, undefined, 3000); // wait up to 5 minutes for kernel change
 	}
+
+	async trustNotebook(): Promise<void> {
+		await this.code.waitAndClick(NotebookToolbar.toolbarSelector);
+
+		let buttons: IElement[] = await this.code.waitForElements(NotebookToolbar.toolbarButtonSelector, false);
+		buttons.forEach(async button => {
+			if (button.className === NotebookToolbar.trustedButtonClass) { // notebook is already trusted
```

Comment:
I'd suggest just doing button.classList.contains('icon-shield') type checks for these. Requiring that the class names all be exactly the same and in the same order is unnecessary and more prone to failure.